controllers/user.go

```diff
@@ -35,6 +36,26 @@ func User(ctx context.Context, w http.ResponseWriter, r *http.Request) (status i
 	}
 }
 
+func GetUser(ctx context.Context, w http.ResponseWriter, r *http.Request) (status int, err error) {
```

Comment:
Yeah but not everybody should see the profile of everybody. The company that invited a user to the challenge may see it, and the user himself.

In the future we have to offer hiding the e-mail address at least.